### PR TITLE
Add documentation to note there are extension changes for oopbas

### PR DESCRIPTION
**Pull request by robmen**
Also add a note that building bundles using wix.exe requires updating to the new `WixToolset.BootstrapperApplications.wixext`.

Fixes wixtoolset/issues#8155

### CODE REVIEW

**Comment by robmen** [comment]
Hmm. The auto-merge didn't wait for approval by an assigned reviewer. That's not right.

File: src/Docusaurus/docs/fivefour/oopbas.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ To take advantage of the breaking change, we took the opportunity to improve the
 * `WixToolset.BalUtil` - renamed to `WixToolset.BootstrapperApplicationApi` to provide the native headers and libraries to communicate with Burn. Also, split out the `WixToolset.WixStandardBootstrapperApplicationFunctionApi` for WixStdBA BAFunctions API.
 * `WixToolset.Mba.Core` - merged the managed libraries into the `WixToolset.BootstrapperApplicationApi` so there is a single package for custom bootstrapper applications.
 * `WixToolset.BextUtil` - renamed to `WixToolset.BootstrapperExtensionApi`.
-* `WixToolset.Bal.wixext` - renamed to `WixToolset.BootstrapperApplications.wixext` but also kept `WixToolset.Bal.wixext` for backwards compatibility.
+* `WixToolset.Bal.wixext` - renamed to `WixToolset.BootstrapperApplications.wixext` but also kept `WixToolset.Bal.wixext` for backwards compatibility with `PackageReferences` in MSBuild. Note: this means that if using `wix.exe` to add extensions you will need to use the new name `WixToolset.BootstrapperApplications.wixext`.
```
Comment (barnson) [inline src/Docusaurus/docs/fivefour/oopbas.md:83]: > `PackageReferences` in MSBuild.
`PackageReferences` in MSBuild projects.

> Note: this means that if using `wix.exe` to add extensions you will need to use the new name `WixToolset.BootstrapperApplications.wixext`.
Note: When using `wix.exe`, you need to use the new name `WixToolset.BootstrapperApplications.wixext`.

File: src/Docusaurus/docs/fivefour/index.md [hunk 2]
```diff
@@ -199,3 +199,5 @@ For more information, see the [_Default feature_ WIP](https://github.com/wixtool
...
 - WixToolset.Netfx.wixext's [DotNetCompatibilityCheck](../schema/netfx/dotnetcompatibilitycheck.md) now sets the specified property to 13 when the requested platform is not compatible with the platform the installer is running on. Thanks to [@apacker1](https://github.com/apacker1) for the [pull request](https://github.com/wixtoolset/wix/pull/459).
 - WixToolset.Util.wixext now has the following `_NODOMAIN` properties from [WixQueryOsWellKnownSID](../tools/wixext/wininfo.md#querywindowswellknownsids): `WIX_ACCOUNT_ADMINISTRATORS_NODOMAIN`, `WIX_ACCOUNT_GUESTS_NODOMAIN`, `WIX_ACCOUNT_LOCALSERVICE_NODOMAIN`, `WIX_ACCOUNT_LOCALSYSTEM_NODOMAIN`, `WIX_ACCOUNT_NETWORKSERVICE_NODOMAIN`, and `WIX_ACCOUNT_USERS_NODOMAIN`. Thanks to [@mistoll](https://github.com/mistoll) for the [pull request](https://github.com/wixtoolset/wix/pull/407).
+
+See [Out-of-process bootstrapper applications](/docs/fivefour/oopbas/) for additional extension changes related to building bundles.
```
Comment (barnson) [inline src/Docusaurus/docs/fivefour/index.md:203]: > applications](/docs/fivefour/oopbas/)
applications](oopbas.md)

**Review by barnson — CHANGES_REQUESTED** [review]
Otherwise lgtm.

**Pull request merged**